tools/Makefile

```diff
@@ -28,10 +14,6 @@ prep-root:
 	@ln -s $(abspath ../webui/react/build/*) build/webui/react
 	@ln -s $(abspath ../harness/dist/*.whl) build/wheels/
 
-.PHONY: run
```

Comment:
Why not leave `make -C tools run` intact for a while?  I think rigidly forcing devs to transition from one tooling system to another one is an unnecessary burden.

I think there should be at least a couple months after I land `make devcluster` and before we remove `make -C tools run`